docs/pages/reference/mesheryctl-commands.md

```diff
@@ -117,13 +117,26 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
       </tr>
     {% endfor %}
   {% endfor %}
-    {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.subcommands %}{% assign subcommand = subcommand_hash[1] %}
+  {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.configure.command %}{% assign subcommand = subcommand_hash[1] %}
+    <tr>
+      <td rowspan=1><a href="{{ site.baseurl }}/reference/mesheryctl/system/config">{{ subcommand.name }}</a></td>
+      <td></td>
+      <td>{{ subcommand.description }}</td>
+    </tr>
+    {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system.configure.flag %}{% assign flag = flag_hash[1] %}
       <tr>
-        <td>{{ subcommand.name }}</td>
-        <td></td>
-        <td>{{ subcommand.description }}</td>
+        <td>{{ flag.flag }}</td>
+        <td>{{ flag.description }}</td>
       </tr>
     {% endfor %}
+  {% endfor %}
```

Comment:
There are two configs now, I'm able to go to the link and check the `mesheryctl system config` page

![image](https://user-images.githubusercontent.com/23498248/119497220-f263f500-bd81-11eb-9e0a-1bb1e167d3ea.png)

Do refer this doc for adding a bit more about the command, https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0

Reply:
Also @Blakelist7 Would you mind changing this to ---> This PR fixes #2940 

If they are not on the same line, when this PR is merged the issue won't be closed

![image](https://user-images.githubusercontent.com/23498248/119498637-75397f80-bd83-11eb-84ef-69e0b30ca20b.png)

Reply:
Done

Reply:
Please have a look @adithyaakrishna